test/e2e/metrics_test.go

```diff
@@ -19,8 +19,12 @@ import (
 )
 
 func testMetrics(t *testing.T) {
+	err := framework.AddToFrameworkScheme(monitoringv1.AddToScheme, &monitoringv1.ServiceMonitor{})
+	require.NoError(t, err)
+
 	t.Run("Windows Exporter configuration validation", testWindowsExporter)
 	t.Run("Prometheus configuration validation", testPrometheus)
+	t.Run("Windows Prometheus rules validation", testWindowsPrometheusRules)
```

Comment:
Is it possible to add a test that actually ensures we are seeing the expected metrics?

Reply:
[TestWindowsExporter](https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/metrics_test.go#L55) tests if we can access the windows metrics provided by the `windows-exporter`. Do we want to test if those metrics are as expected ? One of the [tests](https://github.com/prometheus-community/windows_exporter/blob/master/tools/end-to-end-test.ps1) that `windows-exporter` do is to compare the output from `/metrics` endpoint to a given temp file.

Should testing metrics from `windows-exporter` be included in our repo as well ?

Reply:
Let me state this in another way. You are adding a set of rules where you are mapping metrics exposed by `windows-exporter` to look like Linux metrics so that they work in the console. I was asking if there was a way to test if that is happening correctly.